edx/edx-app-ios
# Add dynamic text type support

### Description

[LEARNER-6794](https://openedx.atlassian.net/browse/LEARNER-6794)

App was not supporting the accessibility feature of dynamic type texting, I added the support for all screens having native elements, now all native elements(UILabel, UITextField, UIButton, UISegmentView) are dynamic type accessible, we can increase the font size from accessibility setting. 

### Notes

### How to test this PR

- Open iPhone Setting >> Accessibility >> Large Text
- Enable Larger Accessibility Sizes
- Slide the font size either toward extra large or extra small
- Open the app, the font size of the landing and other screens should be updated as per setting.
